### ISSUE TITLE
Updates panel list: Show link to view list table of all updates for this post

### ISSUE THREAD

**Issue by coreymckrill** [WordPress/revisions-extended#98]
The Updates list table screen accepts a query parameter to only show updates for a particular parent post, by specifying the post ID, like so: `/wp-admin/edit.php?page=post-updates&p=1`

In the Updates panel in the block editor, beneath the list of updates, it would be nice to add a link with this URL, "See all updates" or something.

Alternatively, perhaps we should consider making the Updates panel more like the built in Revisions panel, and just show something like `3 Updates`, where clicking on that goes to the list table?

**Comment by StevenDufresne** [comment]
> Alternatively, perhaps we should consider making the Updates panel more like the built-in Revisions panel, and just show something like 3 Updates, where clicking on that goes to the list table?

Good question. I'm thinking that the 'update' flow is different from the 'revision' flow in that updates are in active development and if there is an update, there's a much higher likelihood the user wants to work on the 'latest' update... so having the latest one, at least, would be a shorter more intuitive journey. 

I would guess that the other flow, where users are managing multiple updates across multiple posts, would start on the update list page and not require the link backwards???

**Comment by StevenDufresne** [comment]
Maybe we could show the latest update (or the one that will go live) in the `post` editor and then add a new control that mimics the revision on one in the `update` editor to see all?

**Comment by coreymckrill** [comment]
> so having the latest one, at least, would be a shorter more intuitive journey.

Good point. Nice to go straight to the update editor instead of having to go through the list table first. Maybe we could show just the most recently modified update, and then if there are more, show the "See all updates" link?

> ... add a new control that mimics the revision on one in the update editor to see all?

Yeah, that seems like a good idea.
